yt_dlp/extractor/newgrounds.py

```diff
@@ -4,54 +4,81 @@
 
 from .common import InfoExtractor
 from ..utils import (
-    ExtractorError,
     extract_attributes,
     int_or_none,
     parse_duration,
     parse_filesize,
+    parse_count,
```

Comment:
this should be at line 9

Reply:
Ok, but why?

Reply:
`c` comes before `d`
tbh idk if there's guideline to sort the imports but it looks clean and not out of place.

Reply:
Thanks